imap_processing/cdf/config/imap_glows_l1a_variable_attrs.yaml

```
@@ -0,0 +1,381 @@
+# Info found in cdf.defaults.py
+# Info defined from glows_cdf_attrs
+int_fillval: &int_fillval -9223372036854775808
+min_epoch: &min_epoch -315575942816000000
+max_epoch: &max_epoch 3155630469184000000
+
+#Information taken from CDF warnings of missing attributes
+default_attrs: &default_attrs
+  # Resolution is optional?
+  RESOLUTION: ' '
+  DISPLAY_TYPE: no_plot
+  TIME_BASE: J2000
+  # Guess from other l1a sections, but optional?
+  TIME_SCALE: Terrestrial Time
+  # Guess from other l1a sections
+  REFERENCE_POSITION: Rotating Earth Geoid
+  # Guess from other l1a sections
+  UNITS: ns
```

I'd say that this should be unitless by default. 
```suggestion
  UNITS: ' '
```